Makefile

```diff
@@ -22,14 +23,14 @@ build: $(BINDATA)
 	$(GO_BUILD_RECIPE)
 
 # Using "-modtime 1" to make generate target deterministic. It sets all file time stamps to unix timestamp 1
-generate $(BINDATA): $(GOBINDATA_BIN)
+$(BINDATA): $(GOBINDATA_BIN) $(ASSETS)
```

Comment:
So the name of this build target is `pkg/manifests/bindata.go`?

Reply:
Yes.  This will trigger `pkg/manifests/bindata.go` to be re-generated when assets change.

Reply:
ack, just an odd name is all